third_party/com_github_pugixml/workspace.bzl

```diff
@@ -0,0 +1,19 @@
+load("//third_party:repo.bzl", "third_party_http_archive")
+load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
+
+def repo():
+    maybe(
+        third_party_http_archive,
+        name = "com_github_pugixml",
+        urls = [
```

Comment:
This needs to add strip_prefix, and then remove the prefix manipulation from bundled.BUILD.bazel.

   `strip_prefix = "pugixml-1.13",`

Reply:
updated